app.py

```diff
@@ -3,46 +3,14 @@
 from dash import Dash, Input, Output, dcc, html
 import read_data
 import mapping
+import layout
```

Comment:
nitpick: Switch this to `from layout import get_layout`.  It is nicer stylistically.  Then you can remove the `layout.` below.